analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs

```diff
@@ -839,7 +839,7 @@ public void ReferenceEqualsBool()
         {
             bool a, b;
             a = b = true;
-            if (object.ReferenceEquals(a, b)) { }   // FN
+            if (object.ReferenceEquals(a, b)) { }   // Noncompliant
```

Comment:
Is this actually correct? Check the message to see whether this always evaluates to `true` or `false`.